faiss/gpu/test/TestGpuIndexCagra.cu

```diff
@@ -330,6 +330,9 @@ void copyToTest(
                 recall_score.view(),
                 copy_ref_dis_mds_opt,
                 ref_dis_mds_opt);
+        std::cout << "run: " << tries
+                  << ", recall_score: " << *recall_score.data_handle()
+                  << std::endl;
         ASSERT_TRUE(*recall_score.data_handle() > expected_recall);
```

Comment:
Please consider to change this to the line below as well. It should output the values on error which is convenient.

```
ASSERT_GT(*recall_score.data_handle(), expected_recall);
```